### ISSUE TITLE
Output match file

### ISSUE THREAD

**Issue by EZarnosky** [Fribb/MyAnimeList.bundle#43]
Would it be possible for the MAL Bundle to output a .match file to the directory of a matched series (kind of like the LAMDA bundle does for NFO and other files)? I have over 5000 series and my only option for match files would be programmatical.

If this is not possible maybe a separate tool to help by scanning all folders, suggesting matches and upon ok, output a .match file.

**Comment by Fribb** [comment]
Technically speaking, this should be possible. There shouldn't be much of a difference between the agent reading the `.match` file and writing one.

But I would still have some thoughts about this:

1. the `.match` file was implemented that way on purpose to allow other agents (if this should be the case) to use the same thing and not create other match files if they choose to have such a feature. (maybe it would be better to revert back to the `.malmatch` to be more in line with the `.plexmatch` file?!)
2. if the `.match` file is kept as-is, the agent should only change its own value, the mal-id.
3. this should also only be an "either-or" feature. So either you request metadata or you write metadata to the `.match` file. But not both

On the other hand, this feature would be used quite rarely. This might be better implemented as a standalone script that then interfaces with Plex or even a separate agent (like the already mentioned lambda) that does this.

A separate agent would also allow you to run them both in tandem through the Agent Priority list. So you, for example, run the MAL Agent first to search and save the metadata and then the other agent to save the `.match` file.

**Comment by EZarnosky** [comment]
I think a .malmatch would be great. I can't tell you how annoying it is to have a series I got 7 years ago and matched fine is now under a different name at mal and won't match anymore.

I agree after thinking about it, a standalone script might be better, maybe something with a GUI frontend?

I did a recent rebuild of my plex server after a major DB corruption and my anime library is all messed up. I need for force match a lot of series and fix several incorrect matches (5000 series and 1000 movies). a GUI tool (maybe in a spreadsheet format) that provides a best match and then allows me to correct it if need be would be very helpful.

I am sure whatever you might create would be of great use to everyone as the MAL bundle is too.

Any thoughts to a TinyMediaManager plug in? I know it does not create a match file, but it does grab everything else (posters, nfo, trailers, subs, etc.). Not sure if the plugin can be set to create a custom file like a .malmatch
